laboratory-servlet-api-integration/pom.xml

```diff
@@ -163,30 +180,31 @@
                         <configuration>
                             <sources>
                                 <source>src/it/java</source>
+                                <source>src/it/scala</source>
                             </sources>
                         </configuration>
                     </execution>
                 </executions>
             </plugin>
 
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-failsafe-plugin</artifactId>
-                <version>2.6</version>
-                <configuration>
-                    <additionalClasspathElements>
-                        <additionalClasspathElement>src/it/resources</additionalClasspathElement>
-                    </additionalClasspathElements>
-                </configuration>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>integration-test</goal>
-                            <goal>verify</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
+            <!--<plugin>-->
```

Comment:
why? and if for a good reason, can/should it be removed from all the other poms? (and yes we have a ticket for cleaning the poms + removing duplications..)